web/thesauruses/python/3/queues_stacks.json

```diff
@@ -0,0 +1,388 @@
+                "not-implemented": "true"
+
+      },
+      "circular_queue_data_structure": {
+        "name": "Data structure that backs the circular queue",
+                "not-implemented": "true"
+
+      },
+      "priority_queue_import_statement": {
+        "name": "Import statement to add priority queue",
+        "code": [
+          "from queue import PriorityQueue"
+        ]
+      },
+      "priority_queue_data_type": {
+        "name": "Queue data type name",
+        "code": [
+          "queue.PriorityQueue"
+        ]
+      },
+      "priority_queue_create_statement": {
+        "name": "Create a queue",
+        "code": [
+          "queue = PriorityQueue()"
+        ]
+      },
+      "priority_queue_create_copy_statement": {
+        "name": "Create a queue from existing queue",
+        "code": [
+          "new_queue = queue"
+        ]
+      },
+      "priority_queue_create_copy_from_list_statement": {
+        "name": "Create a queue from a list of items",
+        "code": [
+          "from queue import PriorityQueue\nimport itertools\n\nqueue = PriorityQueue()\n\npriorities = [1, 2, 3]\n\nmylist = ['One', 'Two', 'Three']\n\nfor priority, item in list(zip(priorities, mylist)):\n\tqueue.put((priority, item))"
+        ]
+      },
+      "priority_queue_destroy_statement": {
+        "name": "Destroy/delete a queue",
+        "code": [
+          "for item in range(queue.qsize()):\n\tqueue.get()"
+        ]
+      },
+      "priority_queue_data_structure": {
+        "name": "Data structure that backs the priority queue",
+        "code": [
+          "list"
+        ]
+      },
+      "double_ended_queue_import_statement": {
+        "name": "Import statement to add double-ended queue",
+        "code": [
+          "from collections import deque"
+        ]
+      },
+      "double_ended_queue_data_type": {
+        "name": "Queue data type name",
+        "code": [
+          "collections.deque"
+        ]
+      },
+      "double_ended_queue_create_statement": {
+        "name": "Create a queue",
+        "code": [
+          "queue = deque(['One','Two','Three'])"
+        ]
+      },
+      "double_ended_queue_create_copy_statement": {
+        "name": "Create a queue from existing queue",
+        "code": [
+          "new_queue = queue"
+        ]
+      },
+      "double_ended_queue_create_copy_from_list_statement": {
+        "name": "Create a queue from a list",
+        "code": [
+          "mylist = ['One','Two','Three']\nqueue = deque(mylist)"
+        ]
+      },
+      "double_ended_queue_destroy_statement": {
+        "name": "Destroy/delete a queue",
+        "code": [
+          "queue.clear()"
+        ]
+      },
+      "double_ended_queue_data_structure": {
+        "name": "Data structure that backs the simple queue",
+        "code": [
+          "list"
+        ]
+      },
+      "stack_import_statement": {
+        "name": "Import statement to add stacks",
+        "not-implemented": "true"
```

Comment:
I think stacks you can do with a [LIFOqueue](https://docs.python.org/3/library/queue.html) which is essentially a queue but last-in-first-out (which is how a stack works).